src/content/about/metrics.md

```diff
@@ -0,0 +1,85 @@
+
+Worker script metrics can be inspected by time ranges within the last 30 days. The dashboard includes the charts and information described below.
+
+### Invocation statuses
+
+Worker invocation statuses indicate whether a Worker script executed successfully or failed to generate a response in the Workers runtime. Invocation statuses differ from HTTP status codes. In some cases, a Worker script invocation succeeds but does not generate a successful HTTP status because of another error encountered outside of the Workers runtime. Some invocation statuses result in a [Workers error code](/about/tips/debugging/#error-pages-generated-by-workers) being returned to the client.
+
+Worker invocation statuses:
+
+| Invocation status      | GraphQL field        | Workers Error code | Definition                                                               |
+| ---------------------- | -------------------- | ------------------ | ------------------------------------------------------------------------ |
+| Success                | success              |                    | Worker script executed successfully                                      |
+| Client Disconnected    | clientDisconnected   |                    | HTTP client (i.e. the browser) disconnected before the request completed |
+| Script Threw Exception | scriptThrewException | 1101               | Worker script threw an unhandled Javascript exception                    |
+| ¹ Exceeded Resources   | exceededResources    | 1102, 1027         | Worker script exceeded runtime limits                                    |
+| ² Internal Error       | internalError        |                    | Workers runtime encountered an error                                     |
+
+¹ The Exceeded Resources status may appear when the Worker exceeds a [runtime limit](/about/limits). The most common cause is excessive CPU time, but is also caused by a script exceeding startup time or free tier limits.
+
+² The Internal Error status may appear when the Workers runtime fails to process a request due to an internal failure in our system. These errors are not caused by any issue with the Worker code nor any resource limit. While requests with Internal Error status are rare, we expect that some may appear during normal operation. These requests are not counted towards usage for billing purposes. If you notice an elevated rate of requests with Internal Error status, please check www.cloudflarestatus.com.
+
+### Requests
+
+This chart shows historical request counts from the Workers runtime broken down into successful requests, errored requests, and subrequests.
+
+- Total: All incoming requests registered by a Worker script. Requests blocked by [WAF](https://www.cloudflare.com/waf/) or other security features will not count.
+- Successful requests: Success and Client Disconnected invocation statuses.
+- Errored requests: Every other invocation status.
```

Comment:
Saying every other is not useful here. Also nit but make `Errored requests` -> `Errors`  same with successful requests

Reply:
I opted for Successful requests and Errored requests here because Successes and Errors sounds a little weird.  How would you rewrite "every other invocation status"?

Reply:
I put a suggestion earlier,  but can't find it. actually rewriting the status invocations is a startI am not sure what every other one you are talking about